docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -121,7 +121,7 @@ Complete the following steps to upgrade the server:
 1. Run the fhir-installer on a separate server.
```

Comment:
Update lastupdated at top of file.

Reply:
we'll inheret that when i rebase it on the other edits